dali/python/nvidia/dali/external_source.py

```diff
@@ -255,6 +255,17 @@ class ExternalSource():
 .. note::
     To return a batch of copies of the same tensor, use :func:`nvidia.dali.types.Constant`,
     which is more performant.
+
```

Comment:
Sorry for the confusion in the discussion, but I'd actually think those ..notes:: would be better left in the `parallel` section.

Reply:
Adjusted